src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java

```diff
@@ -49,9 +50,8 @@ public void execute_hasTechOptions_shouldAdd() throws IOException {
         //Arrange
         String packagesIncluded = "io.include.*";
         String packagesExcluded = "io.exclude.*";
-        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = new DescribableList<>(JavaOptions.DescriptorImpl.NOOP);
-        technologyOptions.add(new JavaOptions(packagesIncluded, packagesExcluded));
-        PrConfigCommandExecutor executor = createTestedExecutor(technologyOptions);
+        TECH_OPTIONS.add(new JavaOptions(packagesIncluded, packagesExcluded));

```

Comment:
Where do you reset the `TECH_OPTIONS`? I have a feeling it is shared between the tests.

Reply:
Done